cmd/osv-scanner/__snapshots__/main_test.snap

```diff
@@ -3004,8 +3005,9 @@ No issues found
 Scanning local image tarball "../../internal/image/fixtures/test-alpine.tar"
 
 Container Scanning Result (Alpine Linux v3.18):
-Total 2 packages affected by 41 vulnerabilities (2 Critical, 17 High, 14 Medium, 0 Low, 8 Unknown) from 1 ecosystems.
-41 vulnerabilities have fixes available.
+Total 2 packages affected by 41 known vulnerabilities (2 Critical, 17 High, 14 Medium, 0 Low, 8 Unknown) from 1 ecosystem.
+41 vulnerabilities can be fixed.
```

Comment:
Let's keep the original wording here.

```
41 vulnerabilities have fixes available.
```

Reply:
I think the current wording is fine, and it simplifies pluralization handling as discussed in https://github.com/google/osv-scanner/pull/1582#discussion_r1945861563.